app/src/main/java/com/moonpig/mvisample/productpicker/ProductPickerActivity.kt

```diff
@@ -16,16 +22,21 @@ class ProductPickerActivity : AppCompatActivity() {
 
         success.setOnClickListener {
             ProductDetailActivity
-                    .intentForProduct(this, "1")
+                    .intentForProduct(this, catArmourProductId)
                     .launch()
         }
 
         error.setOnClickListener {
             ProductDetailActivity
-                    .intentForProduct(this, "error")
+                    .intentForProduct(this, notExistingProductId)
                     .launch()
         }
     }
 
     private fun Intent.launch() = startActivity(this)
+
+    companion object {
+        private const val catArmourProductId = "1"
```

Comment:
This prompts a wider discussion as a team with how we do constants. Personally I'm a bit old school and like them all CAPITALS_LIKE_THIS, but if we want to go with lower case, that's fine. @tyates thoughts?

Reply:
Ha, I thought about it while typing. 

I guess with Kotlin and data being in general more immutable I don't see why static vals should be screaming, but I am happy to change it to caps snake casing.

Reply:
My gut reaction is that I like UPPER_CONSTS but I don't really have a good justification for that. And when trying to think of one I realised that I don't really know what the difference is, under the hood, between a `const` and a regular `val`? Is it just that the value exist at compile-time?

Reply:
Lowercase. I have decided.

![image](https://user-images.githubusercontent.com/5410731/49883130-ea21d480-fe29-11e8-863d-1f19c57d111c.png)